addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,540 @@
+		LIST_2("CUP_1Rnd_SMOKE_GP25_M")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"Binocular"};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	primaryWeapon[] = {
+		"CUP_arifle_AKS74_GL",
+		"CUP_arifle_AKS74_GL_Early"
+	};
+	sidearmWeapon[] = {"CUP_hgun_PMM"};
+	magazines[] += {
+		LIST_2("CUP_IlumFlareRed_GP25_M"),
+		LIST_2("CUP_1Rnd_SmokeRed_GP25_M"),
+		LIST_3("CUP_12Rnd_9x18_PMM_M")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_RUS_Beret_VDV"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_st138_prc77"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"CUP_arifle_RPK74_45"};
+	secondaryWeapon[] = {};
+	backPack[] = {"CUP_B_CivPack_WDL"};
+	magazines[] = {
+		LIST_16("CUP_45Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class rat : car {
+	displayName = "Rifleman (Anti-Tank)";
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"CUP_srifle_SVD"};
+	scope[] = {"cup_optic_pso_1"};
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+	magazines[] = {
+		LIST_11("CUP_10Rnd_762x54_SVD_M"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_PKM"};
+	secondaryWeapon[] = {"CUP_hgun_PMM"};
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	magazines[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_12Rnd_9x18_PMM_M")
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	secondaryWeapon[] = {};
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M")
+	};
+};
+
+class mmgtl : mmgac {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+	backpackItems[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M")
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_AGS30_Gun_Bag"};
+};
+
+class hmgac : car {
+	displayName = "HMG Ammo Carrier";
+	secondaryWeapon[] = {};
+	backPack[] = {"CUP_B_AGS30_Tripod_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_AGS30_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_Metis"};
```

Comment:
Replace with RPG-7 with heavier AT rounds (PG-7VL) keep metis on HAT

Reply:
Can do. To note: MAT Gunners are not intended to be used in this loadout as the reference document notes that the Anti-Tank Squad only used Metis ATGMs and beyond the grenadier's RPG-7D no other launchers were mentioned.